clang/include/clang/Basic/Attr.td

```diff
@@ -2944,6 +2944,8 @@ def ReqdWorkGroupSize : InheritableAttr {
               ExprArgument<"YDim", /*optional*/1>,
               ExprArgument<"ZDim", /*optional*/1>];
   let Subjects = SubjectList<[Function], ErrorDiag>;
+  let Accessors = [Accessor<"usesOpenCLArgOrdering",
```

Comment:
It's not just argument ordering that's impacted, it's also whether the arguments are required or not. How about `usesOpenCLSemantics` to make it a bit more generic?